extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -56,7 +56,7 @@ public static function onOasisSkinAssetGroups( &$jsAssets ) {
 		if ( self::isCorrectPageType() ) {
 			$jsAssets[] = 'recirculation_js';
 
-			if ( self::canShowDiscussions() ) {
+			if ( self::canShowDiscussions( $wgCityId ) ) {
```

Comment:
seems like this check is being done many times - is there a way to consolidate this or are the use cases all different?